### PR TITLE
Join tables

### PR DESCRIPTION
**Pull request by Timowest12**
In this PR I:
- Created a table named vets,
- Inserted vet data
- inserted specialities data
- inserted visits data
- created multi table queries

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -22,6 +22,15 @@ CREATE TABLE species(
    name VARCHAR(100)
    );
```
Comment (SajjadAhmad14) [inline schema.sql:24]: Please add a new join table named `specializations` with the `species_id` and `vet_id` columns.
Please add a new join table named `visits` with `animal_id`, `vet_id` and `visit_date`.

File: data.sql [hunk 2]
```diff
@@ -70,3 +70,37 @@ UPDATE animals SET owner_id=3 WHERE name IN ('Devimon','Plantmon');
 UPDATE animals SET owner_id=4 WHERE name IN ('Charmander','Squirtle','Blossom');
 --Dean Winchester owns Angemon and Boarmon.
 UPDATE animals SET owner_id=5 WHERE name IN ('Angemon','Boarmon');
+
```
Comment (SajjadAhmad14) [inline data.sql:73]: Please add a project title and project description to README. It looks more professional if it has a project title, description, and set of instructions to run the project.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Timo, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
Git flow is followed. :heavy_check_mark:
PR has a title and summary. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: schema.sql (hunk 1 above)
Comment (Timowest12) [reply schema.sql:24]: Ooh i forgot to add them. Done.

File: data.sql (hunk 2 above)
Comment (Timowest12) [reply data.sql:73]: done

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🎉🍾✔**
Hello @Timowest12 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**